schema/deploy/shipping/views.sql

```diff
@@ -2093,7 +2093,8 @@ create or replace view shipping.genome_submission_metadata_v1 as
           when identifier_set.name in ('collections-kiosks', 'collections-kiosks-asymptomatic') then 'Shelters'
           when identifier_set.name in ('collections-school-testing-home', 'collections-school-testing-observed') then 'Snohomish Schools'
           when identifier_set.name in ('collections-seattleflu.org') then 'SCH'
-          when identifier_set.name in ('collections-uw-home', 'collections-uw-observed', 'collections-uw-tiny-swabs') then 'HCT'
+          when identifier_set.name in ('collections-uw-home', 'collections-uw-observed', 'collections-uw-tiny-swabs',
+                                       'collections-uw-tiny-swabs-home', 'collections-uw-tiny-swabs-observed') then 'HCT'
```

Comment:
Do we want to leave the pilot study records `collections-uw-tiny-swabs` here or drop them?

Reply:
My thinking was to keep them in the genome metadata mapping for completeness. I don't know whether it's likely they would sequence any of the samples from the pilot swabs, but seemed possible